### ISSUE TITLE
input dimension and model dimension can be different

### ISSUE THREAD

**Issue by yzmyyff** [lucidrains/voicebox-pytorch#7]
The input of the audio model is 80-dim log mel in the paper.
The model dimension is a hyperparameter that takes different values in different experiments.
But in our impl these two values are merged to
```
VoiceBox(..., dim=, ...)
```
Can they be separated?

**Comment by lucidrains** [comment]
i'm currently going the encodec / voco route for starters

however, if you want to PR in the log model encoder / hifigan decoder, as in the paper, i can look into disentangling the dimensions earlier

**Comment by yzmyyff** [comment]
Okay, I'll look into it

**Comment by lucidrains** [comment]
@yzmyyff oh i meant the encoding and decoding logic, like [this](https://github.com/lucidrains/voicebox-pytorch/blob/main/voicebox_pytorch/voicebox_pytorch.py#L326)

**Comment by lucidrains** [comment]
@yzmyyff actually your PR looks good! thank you!
